ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -1553,15 +1647,6 @@ protected static void endToEndReplicationWithMultiNodeSinglePartitionTest(String
 
       // get the data node to inspect replication tokens on
       DataNodeId dataNodeId = clusterMap.getDataNodeId("localhost", interestedDataNodePortNumber);
-      // read the replica file and check correctness
```

Comment:
why did you remove this comment?

Reply:
I think it is hard to keep it up to date + the `expectedTokenSize` is now calculated programmatically (it used to be a constant before)